sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -542,6 +542,7 @@ public abstract partial class RequestContent : System.Net.ClientModel.Core.Pipel
         public static Azure.Core.RequestContent Create(byte[] bytes) { throw null; }
         public static Azure.Core.RequestContent Create(byte[] bytes, int index, int length) { throw null; }
         public static Azure.Core.RequestContent Create(System.IO.Stream stream) { throw null; }
+        public static Azure.Core.RequestContent Create(System.Net.ClientModel.Core.PipelineContent content) { throw null; }
```

Comment:
Per offline discussion, I would like to remove this API prior to review with the BCL team and am looking to see if I can spot an easy way to do it now, and will report back.